doc/nrf/drivers/wifi/nrf700x/nrf700x.rst

```diff
@@ -83,14 +83,14 @@ The nRF700X Wi-Fi driver has the following configuration options:
 API documentation
 *****************
 
-After the nrf700x driver has been initialized, the application will see it as an Ethernet interface.
+After the nRF70 Series driver has been initialized, the application will see it as an Ethernet interface.
 To use the Ethernet interface, the application can use `Zephyr Network APIs`_.
 
 Low-level API documentation
 ***************************
 
-The nRF700X Wi-Fi driver provides a low-level API for use cases where the application needs to access the nRF700X device directly.
-This is typically intended for customers who want to use the nRF700X device in a different platform than Zephyr.
+The nRF70 Series Wi-Fi driver provides a low-level API for use cases where the application needs to access the nRF70 series device directly.
+This is typically intended for customers who want to use the nRF70 series device in a different platform than Zephyr.
```

Comment:
```suggestion
This is typically intended for customers who want to use the nRF70 Series device in a different platform than Zephyr.
```

Reply:
There is one issue happened with my branch, need to delete. I will open new PR with these changes.